tests/integration/server/server_test.go

```diff
@@ -0,0 +1,37 @@
+package server
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/config"
+	"sigs.k8s.io/aws-iam-authenticator/pkg/testutils"
+)
+
+func TestServer(t *testing.T) {
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+
+	adminClient, execClient := testutils.StartAuthenticatorTestFramework(t, stopCh, testutils.AuthenticatorTestFrameworkSetup{
+		ModifyAuthenticatorServerConfig: func(*config.Config) {},
+		AuthenticatorClientBinaryPath:   authenticatorBinaryPath,
+	})
+
+	t.Log("Creating aws-auth")
+	_, err := adminClient.CoreV1().ConfigMaps("kube-system").Create(context.TODO(), &v1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{Name: "aws-auth"},
+		Data:       map[string]string{"mapRoles": fmt.Sprintf("    - rolearn: %s\n      username: system:node:{{EC2PrivateDNSName}}\n      groups:\n        - system:bootstrappers\n        - system:nodes", roleARN)},
+	}, metav1.CreateOptions{})
+	if err != nil {
+		t.Fatalf("test failed: %v\n", err)
```

Comment:
golang will tell me the test failed, no need to prepend error with "test failed" . context would be more helpful , like "error creating configmap" or something.